<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="task1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ms teamsUI評価　期限 2020年5月25日 23:59　　　期限を経過</string>
    </property>
   </widget>
   <widget class="QPushButton" name="task2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>UI実装の提出　期限 2020年6月29日 23:59</string>
    </property>
   </widget>
   <widget class="QPushButton" name="task3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>1/23 出席確認</string>
    </property>
   </widget>
   <widget class="QPushButton" name="past_assignment">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>550</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>過去の課題をさらに表示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="not_submitted">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>＞ 未提出 ( 2 )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitted">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>＞ 提出済み ( 1 )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="team1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>＞ UI/UX論</string>
    </property>
   </widget>
   <widget class="QPushButton" name="team2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>＞ PM演習</string>
    </property>
   </widget>
   <widget class="QPushButton" name="not_submitted_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>＞ 提出済み ( 1 )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="task3_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>レポート1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="past_assignment_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>過去の課題をさらに表示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="past_assignment_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>過去の課題をさらに表示</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
